sw/host/opentitanlib/src/util/voltage.rs

```diff
@@ -46,7 +46,7 @@ mod tests {
     fn test_conversions() -> Result<()> {
         assert_eq!(Voltage(2.5).as_volts(), 2.5);
         assert_eq!(Voltage(2.5).as_millivolts(), 2500);
-        assert_eq!(Voltage(3.141592654).as_millivolts(), 3141);
+        assert_eq!(Voltage(3.1173).as_millivolts(), 3117);
```

Comment:
Why did this change? Did we want to check that things were being rounded down, instead of rounding to the nearest millivolt as before?

Reply:
This changed because clippy thinks 3.14159... is too close to the constant PI to be written out explicitly.  There is no rounding in the conversion, so the particular float value used is of no consequence.